# awesome-expectations
Examples of academic labs with explicitly stated expectations of themselves and their mentees in the form of written lab manuals. In general, this includes:
- Lab manual with expectations of Masters' and PhD students and Postdocs
- Work-life balance philosophies
- Code of Conduct with anti-harassment policies
- Culture of sharing, typically "open science"-focused, especially publicly sharing code and data


## Cellular Biology

* Avasthi, Prachee at University of Kansas Medical Center. [Lab Website](http://www.avasthilab.org/). [Lab Manual](http://www.avasthilab.org/lab-policies-and-tips/). [Twitter](https://twitter.com/PracheeAC)
  > Our lab studies mechanisms that regulate the normal and pathological functions of the nearly ubiquitous organelle, the cilium. This highly conserved cellular antenna, also known as a flagellum, requires coordination of the cell cycle, cytoskeletal dynamics, and intracellular trafficking for structural maintenance and signal transduction. Due to the role of the cilium in essential functions in nearly all human cells, abnormalities result in a wide range of diseases, termed ciliopathies. In fact, ciliary signaling is now understood to play a role in very common diseases such as diabetes and cancer and may provide new avenues for therapeutic intervention for these devastating disorders. 

  
## Computational Biology

* Brown, C Titus at the Population Health and Reproduction department at University of California, Davis. [Lab Website](http://ivory.idyll.org/lab/index.html). [Lab Manual](http://ivory.idyll.org/lab/work-guidelines.html). [Twitter](https://twitter.com/ctitusbrown). [GitHub](https://github.com/dib-lab/)
  > The DIB Lab tackles questions surrounding biological data analysis, data integration, and data sharing. Our primary interest is in genomic, transcriptomic, and metagenomic sequence analysis.
  > In brief,
  > 
  > - the lab is the primary developer of the khmer software, for faster and more efficient sequence analysis of high-throughput sequencing data.
  > - we run quite a bit of training in data-intensive biology.
  > - we coordinate training and communication for the NIH-funded Data Commons Pilot Phase.
  > - the lab is located at the University of California, Davis in the School of Veterinary Medicine.

* Coelho, Luis Pedro at the Fudan University (Shanghai, China). [Lab Website](http://big-data-biology.org/). [Lab Manual](http://big-data-biology.org/manual/). [Twitter](https://twitter.com/luispedrocoelho).
  > We are interested both in developing novel computational methods and in
  > applying them to large scale problems. Our focus is on the _global
  > microbiome_ and in exploiting publicly available data to gain understanding
  > into microbial ecosystems.
  >
  > _Lab motto_: "Ever tried. Ever failed. No matter. Try Again. Fail again.
  > Fail better. â€” S. Beckett, in other words, **failing is the job**.
   
  
  
* Greene, Casey S at the Department of Systems Pharmacology and Translational Therapeutics at the University of Pennsylvania. [Lab Website](http://greenelab.com/). [Lab Manual](https://github.com/greenelab/onboarding/blob/master/onboarding.md). [Twitter](https://twitter.com/greenescientist). [GitHub](https://github.com/greenelab/)
  > We view our core purpose as the development of methodological advances and integrative systems that make analysis of big data, particularly gene expression data, as routine in wet-bench biology labs as PCR. To accomplish this, we will write good code, perform solid and reproducible analyses, and disseminate our results widely through approachable publications and webservers. We recognize that trust, both in the process and in our results, is of primary importance to the biologists that use our methods and webservers. Therefore, we strive to make our source code as open and accessible as possible. When we submit papers, we expect that the analytical code behind those papers will be something that we can be proud of. To these ends, we will provide reviewers and the scientific community with all source code required to generate figures in the paper that result from computational analyses.
  
## Molecular Biology

* DePace, Angela at the Systems Biology Department at Harvard Medical School. [Lab Website](https://depace.med.harvard.edu/). [Lab Manual](https://depace.med.harvard.edu/?page_id=408). [Twitter](https://twitter.com/depacelab).
  > Our long-term goal is to understand how regulatory DNA dictates transcriptional network behavior and, ultimately, organismal phenotype and evolution. Our approach is mechanistically motivated: we believe that understanding the molecular mechanisms that drive transcription will lead to models of gene regulation that can predict the functional consequences of regulatory sequence changes and guide production of new types of regulatory circuits.




